packages/cashier/src/types/shared/websocket.types.ts

```diff
@@ -1,5 +1,37 @@
+import { Balance } from '@deriv/api-types';
+import { TMT5LoginAccount } from 'Types';
+
 export type TServerError = {
     code: string;
     message: string;
     details?: { [key: string]: string };
 };
+
+type TWebSocketCall = {
+    cashier: (action: any, parameters: any) => Promise<any>;
```

Comment:
are you sure of using here  and below 'any'? there is no sense of typescript so

Reply:
Changed.